tests/integration/testers/cephfs-client/charmcraft.yaml

```
@@ -0,0 +1,24 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: cephfs-client-mock
+type: charm
+
+title: CephFS Client Mock.
+
+summary: Test charm for CephFS server proxy.
+
+description: |
+  Mock charm to test the CephFS server proxy charm. 
+
+bases:
+  - build-on:
+      - name: ubuntu
+        channel: "22.04"
+    run-on:
+      - name: ubuntu
+        channel: "22.04"
+
+requires:
+  cephfs-share:
+    interface: cephfs_share
+    limit: 1
```

[nit]: Missing a newline at the end of the file:

```suggestion
    limit: 1
    
```